modules/oe_theme_content_tender/src/Plugin/ExtraField/Display/TenderStatusExtraField.php

```diff
@@ -88,7 +88,11 @@ public function viewElements(ContentEntityInterface $entity) {
     if ($status === TenderNodeWrapperInterface::STATUS_OPEN) {
       $cacheable->addCacheTags($this->cacheTagGenerator->generateTags($entity->getDeadlineDate()->getPhpDateTime()));
     }
-    $build['#markup'] = $entity->getStatusLabel();
+    $build = [
+      '#theme' => 'oe_theme_content_tender_status',
+      '#label' => $entity->getStatusLabel(),
+      '#name' => $status,
```

Comment:
`#name` isn't used in the template. Do we need extra template?

Reply:
Name is used here to set the background: https://github.com/openeuropa/oe_theme/blob/79c2e86dddc942835e8c7accc89ea6a2d65dbe6c/modules/oe_theme_content_tender/templates/oe-theme-content-tender-label-status.html.twig#L9

Reply:
oe-theme-content-tender-label-status.html.twig is another template.

Reply:
https://github.com/openeuropa/oe_theme/blob/3a203d312df0971ae6287ca0c724817f04a0271d/modules/oe_theme_content_tender/src/Plugin/ExtraField/Display/TenderStatusExtraField.php is the parent class for https://github.com/openeuropa/oe_theme/blob/3a203d312df0971ae6287ca0c724817f04a0271d/modules/oe_theme_content_tender/src/Plugin/ExtraField/Display/TenderLabelStatusExtraField.php and this one needs `#name' because it uses oe-theme-content-tender-label-status.html.twig, maybe we can remove it from TenderStatusExtraField.php and define it only on  TenderLabelStatusExtraField.php ?